### PR TITLE
Privacy Sandbox landing page

### PR DESCRIPTION
**Pull request by heyawhite**
Adds:
- PS landing in new style
- Macro for expanded landing section, with a spot for additional links.
- Macro for related link list
- Macro for use cases landing section, with up to two ordered lists, in addition to "Other docs"
- Associated CSS customization

https://deploy-preview-3233--developer-chrome-com.netlify.app/privacy-sandbox

To address: the workbox color is set in the header card and is not a variable. 

~**Open concerns, not yet the addressed**~
~- Redirect from `/privacy-sandbox` -> `/docs/privacy-sandbox`~

### CODE REVIEW

File: site/en/privacy-sandbox/index.njk [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      description: Read a high-level explanation of the API, use cases, features, and intended audience.
+    - url: /docs/privacy-sandbox/attribution-reporting-experiment
+      title: Experiment and participate
+      description: A step-by-step guide to implement and test the API. Play with demo code and share your feedback.
+    - url: /docs/privacy-sandbox/attribution-reporting-updates
+      title: API updates
+      description: A quick reference to how the proposal has changed to address community feedback.
+    - url: /docs/privacy-sandbox/attribution-reporting/summary-reports
+      title: Summary reports
+      description: Measure ad conversions aggregated across users, without revealing individual data.
```
Comment (heyawhite) [inline site/en/privacy-sandbox/index.njk:61]:
```suggestion
    - url: /docs/privacy-sandbox/attribution-reporting/system-overview
      title: System overview
      description: High-level overview of connected services for Attribution Reporting, aimed at technical decision makers.
    - url: /docs/privacy-sandbox/attribution-reporting
      title: Attribution Reporting
      description: Read a high-level explanation of the API, use cases, features, and intended audience.
    - url: /docs/privacy-sandbox/attribution-reporting-experiment
      title: Experiment and participate
      description: A step-by-step guide to implement and test the API. Play with demo code and share your feedback.
    - url: /docs/privacy-sandbox/attribution-reporting-updates
      title: API updates
      description: A quick reference to how the proposal has changed to address community feedback.
```

File: site/en/privacy-sandbox/index.njk [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    - url: /docs/privacy-sandbox/fledge-experiment
+      title: Experiment and participate
+      description: A step-by-step guide to implement and test the API. Play with demo code and share your feedback.  
+  measure_ads:
+    - url: /docs/privacy-sandbox/attribution-reporting/system-overview
+      title: System overview
+      description: High-level overview of connected services for Attribution Reporting, aimed at technical decision makers.
+    - url: /docs/privacy-sandbox/attribution-reporting
+      title: Attribution Reporting
+      description: Read a high-level explanation of the API, use cases, features, and intended audience.
```
Comment (heyawhite) [inline site/en/privacy-sandbox/index.njk:55]:
```suggestion
    - url: /docs/privacy-sandbox/attribution-reporting
      title: Attribution Reporting
      description: Read a high-level explanation of the API, use cases, features, and intended audience.
    - url: /docs/privacy-sandbox/attribution-reporting/system-overview
      title: System overview
      description: High-level overview of connected services for Attribution Reporting, aimed at technical decision makers.
```

File: site/en/privacy-sandbox/index.njk [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+    - url: /docs/privacy-sandbox/fledge-experiment
+      title: Experiment and participate
+      description: A step-by-step guide to implement and test the API. Play with demo code and share your feedback.  
+  measure_ads:
+    - url: /docs/privacy-sandbox/attribution-reporting
+      title: Attribution Reporting
+      description: Read a high-level explanation of the API, use cases, features, and intended audience.
+    - url: /docs/privacy-sandbox/attribution-reporting/system-overview
+      title: System overview
+      description: High-level overview of connected services for Attribution Reporting, aimed at technical decision makers.
```
Comment (heyawhite) [inline site/en/privacy-sandbox/index.njk:55]:
```suggestion
    - url: /docs/privacy-sandbox/attribution-reporting
      title: Attribution Reporting
      description: Read a high-level explanation of the API, use cases, features, and intended audience.
    - url: /docs/privacy-sandbox/attribution-reporting/system-overview
      title: System overview
      description: Review how services connect end-to-end for Attribution Reporting.
```

**Review by mihajlija — APPROVED** [review]

**Review by rachelandrew — APPROVED** [review]
LGTM - excited to see these go live!

**Pull request merged**